doc/whats_new/v0.21.rst

```diff
@@ -542,6 +542,12 @@ Multiple modules
 - |Efficiency| Memory copies are avoided when casting arrays to a different
   dtype in multiple estimators. :issue:`11973` by :user:`Roman Yurchak
   <rth>`.
+- Add estimators tags: these are annotations of estimators that allow
```

Comment:
``- |MajorFeature|``?

Reply:
Could just be feature in terms of user impact. It's a major achievement on our part